rest/route/project_events.go

```diff
@@ -0,0 +1,99 @@
+package route
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/evergreen-ci/gimlet"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+)
+
+func makeFetchProjectEvents(sc data.Connector) gimlet.RouteHandler {
+	return &projectEventsGet{sc: sc}
+}
+
+type projectEventsGet struct {
+	Timestamp time.Time
+	Limit     int
+	Id        string
+	sc        data.Connector
+}
+
+func (h *projectEventsGet) Factory() gimlet.RouteHandler {
+	return &projectEventsGet{
+		Timestamp: time.Now(),
+		Limit:     10,
+		Id:        "",
+		sc:        h.sc,
+	}
+}
+
+func (h *projectEventsGet) Parse(ctx context.Context, r *http.Request) error {
+	var err error
+
+	h.Id = gimlet.GetVars(r)["project_id"]
+
+	vals := r.URL.Query()
+	k, ok := vals["ts"]
+	if ok && len(k) > 0 {
+		h.Timestamp, err = time.Parse(time.RFC3339, k[0])
+		if err != nil {
+			return errors.Wrap(err, "problem parsing time as RFC-3339")
+		}
+	}
+
+	h.Limit, err = getLimit(vals)
+	if err != nil {
+		return errors.WithStack(err)
+	}
+
+	return errors.WithStack(err)
+}
+
+func (h *projectEventsGet) Run(ctx context.Context) gimlet.Responder {
+	resp := gimlet.NewResponseBuilder()
+
+	events, err := h.sc.GetProjectEventLog(h.Id, h.Timestamp, h.Limit+1)
+	if err != nil {
+		return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "database error"))
+	}
+
+	lastIndex := len(events)
+	if len(events) > h.Limit {
+		lastIndex = h.Limit
+		err = resp.SetPages(&gimlet.ResponsePages{
+			Next: &gimlet.Page{
+				BaseURL:         h.sc.GetURL(),
+				KeyQueryParam:   "ts",
+				LimitQueryParam: "limit",
+				Relation:        "next",
+				Key:             events[h.Limit-1].Timestamp.Format(time.RFC3339),
```

Comment:
I think there is a timestamp format handler. (maybe.) 

we might also panic in a case where the limit is 1. 

are the query param names consistent with our use elsewhere?

Reply:
This seems to be how we format timestamps elsewhere.
I don't see why it will panic where limit==1. If it's0 it might, but this is handled by getLimit()
The query parameters are consistent.